### ISSUE TITLE
Bug with overwriting checkpoint training hyperparameters when finetuning

### ISSUE THREAD

**Issue by roedoejet** [EveryVoiceTTS/EveryVoice#41]
@davidguzmanr reports that the learning rate cannot be overwritten when fine-tuning (ie loading from a checkpoint). Some hyperparameters should not be able to be overwritten, but all those related to training should be able to.
